src/keboola_mcp_server/tools/doc.py

```
@@ -31,6 +33,53 @@ class DocsAnswer(BaseModel):
     source_urls: list[str] = Field(description='List of URLs to the sources of the answer.')
 
 
+class GlobalSearchAnswer(BaseModel):
+    """An answer to a global search query."""
+
+    class Item(BaseModel):
+        """An item found in the global search."""
+
+        name: str = Field(description='The name of the item.')
+        id: str = Field(description='The id of the item.')
+        type: GlobalSearchTypes = Field(description='The type of the item.')
+        created: datetime = Field(description='The date and time the entity was created.')
+        additional_info: dict[str, Any] = Field(description='Additional information about the item.')
+
+        @classmethod
+        def from_api_response(cls, item: GlobalSearchResponse.Item) -> 'GlobalSearchAnswer.Item':
+            """Creates an Item from the API response."""
+            add_info = {}
+            if item.type == 'table':
+                bucket_info = item.full_path.get('bucket', {})
+                add_info['bucket_id'] = bucket_info.get('id', 'unknown')
```

These default values look suspicious. I expect that if you are able to get `bucket_info` it is guaranteed to have an 'id' field.

---

But if that field should be there and is missing, then we should throw an exception, because it's a wrong API response.